neuro-cli/src/neuro_cli/blob_storage.py

```diff
@@ -408,7 +427,7 @@ async def cp(
                 param_type="argument", param_hint='"SOURCES..."'
             )
         sources = *sources, destination
-        target_dir = parse_blob_or_file_resource(target_directory, root)
+        target_dir = target_directory
```

Comment:
Removing `target_dir` and `dst` variables for the sake of reusing already existing `target_directory` and `destination` makes the code a bit more readable and understandable.